src/main/groovy/org/gradle/android/workarounds/ZipMergingTaskWorkaround.groovy

```diff
@@ -0,0 +1,27 @@
+package org.gradle.android.workarounds
+
+import com.android.build.gradle.tasks.ZipMergingTask
+import org.gradle.android.AndroidIssue
+import org.gradle.api.Project
+
+/**
+ * Disables caching of ZipMergingTask which is mostly disk bound and
+ * unlikely to provide positive performance benefits.
+ */
+@AndroidIssue(introducedIn = "3.5.4", fixedIn = [], link="https://issuetracker.google.com/issues/200002454")
```

Comment:
```suggestion
@AndroidIssue(introducedIn = "3.5.0", fixedIn = [], link="https://issuetracker.google.com/issues/200002454")
```
Just in case we have some customers using the cache fix plugin with 3.5.0!